include/El/core/DistGraph.hpp

```diff
@@ -11,6 +11,206 @@
 #ifndef EL_CORE_DISTGRAPH_HPP
 #define EL_CORE_DISTGRAPH_HPP
 
-#include <El/core/DistGraph/decl.hpp>
```

Comment:
In practice there is a lot of movement between the `impl.hpp` and an associated source file. For `DistMatrix`, there have been at least five such movements. If nothing else, consistency is important.